pkg/controllers/rollout/controller_test.go

```diff
@@ -225,7 +228,111 @@ func TestReconcilerUpdateBindings(t *testing.T) {
 		toBeUpgradedBinding        []*fleetv1beta1.ClusterResourceBinding
 		wantErr                    bool
 	}{
-		// TODO: Add negative test cases with fake client
+		"test update binding with no latestResourceSnapshotName": {
```

Comment:
what does this test?

Reply:
Was using this to test if this would cause an issue when updating the ResourceSnapShotName field in the binding. I can delete it.